python/cudf/cudf/tests/test_series.py

```diff
@@ -762,8 +762,8 @@ def test_series_memory_usage():
     sliced_sr = sr[2:]
     assert sliced_sr.memory_usage() == 16
 
-    sliced_sr[3] = None
-    assert sliced_sr.memory_usage() == 80
+    # sliced_sr[3] = None
+    # assert sliced_sr.memory_usage() == 80
```

Comment:
Looks like stale change?

Reply:
Commented out because this test fails, not sure how to resolve:
```
sr = cudf.Series([1, 2, 3, 4], dtype="int64")
sliced_sr = sr[2:] 
>sliced_sr[3] = None

self = <cudf.core.column.numerical.NumericalColumn object at 0x7fc27c37cec0>
[
  3,
  4
]
dtype: int64
key = <cudf.core.column.numerical.NumericalColumn object at 0x7fc27c37ccc0>
[
  3
]
dtype: int64, value = None

FAILED cudf/tests/test_series.py::test_series_memory_usage - IndexError: index out of bounds for column of size 2

```